src/journal/journald-server.c

```diff
@@ -144,7 +144,7 @@ static int cache_space_refresh(Server *s, JournalStorage *storage) {
 
         ts = now(CLOCK_MONOTONIC);
 
-        if (space->timestamp + RECHECK_SPACE_USEC > ts)
+        if (space->timestamp && space->timestamp + RECHECK_SPACE_USEC > ts)
```

Comment:
Hmm, we generally try to avoid relying on C's degrade-to-bool feature for numeric variables. Hence I think in this case we should really write `space->timesamp != 0` or so to see if it is set...

(Also see CODING_STYLE)